lib/windows.js

```diff
@@ -123,8 +165,22 @@ function getProcessPath(processHandle) {
 	const processFileNameBufferClean = ref.reinterpretUntilZeros(processFileNameBuffer, wchar.size);
 	// Get process file path as a string
 	const processPath = wchar.toString(processFileNameBufferClean);
+	let processName = path.basename(processPath);
 
-	return processPath;
+	const unused = ref.alloc('uint32', 1);
+	const infoSize = version.GetFileVersionInfoSizeW(processFileNameBufferClean, unused);
+	if (infoSize !== 0) {
+		// Get data from the FileVersionInfo struct
+		const block = new Buffer.alloc(infoSize);
+		if (version.GetFileVersionInfoW(processFileNameBufferClean, 0, infoSize, block)) {
+			// Get file description from the FileVersionInfo struct
+			const langId = getLanguageId(block);
+			const fileDescription = getVersionInfoForCodePage(block, hexString(langId));
+			processName = fileDescription.trim() || processName;
```

Comment:
`fileDescription` can be undefined for some apps. For me it crashed when opening Windows Terminal or Snipping tool.

```
D:\projects\active-win\node_modules\.pnpm\ffi-napi@4.0.3\node_modules\ffi-napi\lib\_foreign_function.js:61      
    bindings.ffi_call(cif, funcPtr, result, argsList);
             ^

Error: TypeError: Cannot read properties of undefined (reading 'trim')
    at Object.proxy [as EnumChildWindows] (D:\projects\active-win\node_modules\.pnpm\ffi-napi@4.0.3\node_modules\ffi-napi\lib\_foreign_function.js:61:14)    at getSubWindowRealProcessNameAndPath (D:\projects\active-win\lib\windows.js:143:9)
    at Function.windows [as sync] (D:\projects\active-win\lib\windows.js:236:32)
    at Function.module.exports.sync (D:\projects\active-win\index.js:29:38)
```